apache-template.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
```

Comment:
do we need them to be a rust file? can we do away with simple text file?

Reply:
No need to be rust. Can be txt. Will change this.